run_all_clients.py

```diff
@@ -0,0 +1,14 @@
+"""
+Run all clients for map - maps/test_game.json.
```

Comment:
Achjo, ale žadná taková mapa neexistuje! Navíc tento skript nemá nic společného s výběrem mapy...

Reply:
Až mi schvalite PR tak existovat bude. 😂  a je to pravda že k jiné mapě ti to bude k ničemu. Tohle je čistě pro tu jednu testovací a cely ten soubor se bude upravovat a nebo se nakonec taky smaže.